packages/shared/src/components/tooltips/ChangelogTooltip.module.css

```diff
@@ -0,0 +1,17 @@
+.changelogTooltipArrow {
```

Comment:
As much as possible we wanted to sway away from using module CSS since these values are available on tailwind (let me write this down too on confluence). Also, if you need help navigating how to use this tooltip component, as there is no guideline to it yet, I'd be glad to have a coding session with you.

Reply:
Replace with tailwind classes